clayui.com/gatsby-browser.js

```diff
@@ -7,12 +7,18 @@
 // (https://github.com/gatsbyjs/gatsby/issues/6759)
 export const onClientEntry = () => {
 	if (process.env.GATSBY_CLAY_NIGHTLY === 'true') {
-		const isNullOrTrue = (val) => val === 'true' || val === null;
+		const isNullOrTrue = val => val === 'true' || val === null;
 
-		const showAtlas = isNullOrTrue(localStorage.getItem('clay.showAtlas'));
-		const showSiteCss = isNullOrTrue(
-			localStorage.getItem('clay.showSiteCss')
-		);
+		let showAtlas = true;
+		let showSiteCss = true;
+
+		try {
+			showAtlas = isNullOrTrue(localStorage.getItem('clay.showAtlas'));
+
+			showSiteCss = isNullOrTrue(
+				localStorage.getItem('clay.showSiteCss')
+			);
+		} catch {}
```

Comment:
@kresimir-coko I think this should fix the localstorage issue when it is disabled